x/concentrated-liquidity/pool_test.go

```diff
@@ -1,111 +1,105 @@
-	require.NoError(t, err)
-	require.Equal(t, sdk.NewDec(663944647), amountIn.Amount.ToDec())
+	s.Require().NoError(err)
+	s.Require().Equal(sdk.NewDec(663944647), amountIn.Amount.ToDec())
 
 	// test asset a to b logic
 	tokensOut = sdk.NewCoin("usdc", sdk.NewInt(4199999999))
 	tokenInDenom = "eth"
 	swapFee = sdk.NewDecWithPrec(2, 2)
 
 	amountIn, err = pool.CalcInAmtGivenOut(ctx, tokensOut, tokenInDenom, swapFee)
-	require.NoError(t, err)
-	require.Equal(t, sdk.NewDec(821721), amountIn.Amount.ToDec())
+	s.Require().NoError(err)
+	s.Require().Equal(sdk.NewDec(821722), amountIn.Amount.ToDec())
 }
 
-func TestSetInitialPoolDenoms(t *testing.T) {
-	poolDenoms := []string{"eth", "usdc"}
-	pool, err := cl.NewConcentratedLiquidityPool(1, poolDenoms)
-	require.NoError(t, err)
-	require.Equal(t, pool.Token0, poolDenoms[0])
-	require.Equal(t, pool.Token1, poolDenoms[1])
-
-	newPoolDenoms := []string{"axel", "osmo"}
-	err = pool.SetInitialPoolDenoms(newPoolDenoms)
-	require.NoError(t, err)
-	require.Equal(t, pool.Token0, newPoolDenoms[0])
-	require.Equal(t, pool.Token1, newPoolDenoms[1])
-
-	unorderedPoolDenoms := []string{"usdc", "eth"}
-	err = pool.SetInitialPoolDenoms(unorderedPoolDenoms)
-	require.NoError(t, err)
-	require.Equal(t, pool.Token0, unorderedPoolDenoms[1])
-	require.Equal(t, pool.Token1, unorderedPoolDenoms[0])
-
-	tooManyPoolDenoms := []string{"usdc", "eth", "osmo"}
-	err = pool.SetInitialPoolDenoms(tooManyPoolDenoms)
-	require.Error(t, err)
-
-	tooFewPoolDenoms := []string{"usdc"}
-	err = pool.SetInitialPoolDenoms(tooFewPoolDenoms)
-	require.Error(t, err)
-
-	sameDenoms := []string{"usdc", "usdc"}
-	err = pool.SetInitialPoolDenoms(sameDenoms)
-	require.Error(t, err)
-
-}
+// func (s *KeeperTestSuite)TestOrderInitialPoolDenoms(t *testing.T) {
+// 	pool, err := s.App.ConcentratedLiquidityKeeper.CreateNewConcentratedLiquidityPool(s.Ctx, 1, "eth", "usdc", sdk.NewInt(0), sdk.NewInt(0))
+// 	require.NoError(t, err)
+// 	require.Equal(t, pool.Token0, poolDenoms[0])
+// 	require.Equal(t, pool.Token1, poolDenoms[1])
+
+// 	newPoolDenoms := []string{"axel", "osmo"}
+// 	err = pool.SetInitialPoolDenoms(newPoolDenoms)
+// 	require.NoError(t, err)
+// 	require.Equal(t, pool.Token0, newPoolDenoms[0])
+// 	require.Equal(t, pool.Token1, newPoolDenoms[1])
+
+// 	unorderedPoolDenoms := []string{"usdc", "eth"}
+// 	err = pool.SetInitialPoolDenoms(unorderedPoolDenoms)
+// 	require.NoError(t, err)
+// 	require.Equal(t, pool.Token0, unorderedPoolDenoms[1])
+// 	require.Equal(t, pool.Token1, unorderedPoolDenoms[0])
+
+// 	tooManyPoolDenoms := []string{"usdc", "eth", "osmo"}
+// 	err = pool.SetInitialPoolDenoms(tooManyPoolDenoms)
+// 	require.Error(t, err)
+
+// 	tooFewPoolDenoms := []string{"usdc"}
+// 	err = pool.SetInitialPoolDenoms(tooFewPoolDenoms)
+// 	require.Error(t, err)
+
+// 	sameDenoms := []string{"usdc", "usdc"}
+// 	err = pool.SetInitialPoolDenoms(sameDenoms)
+// 	require.Error(t, err)
+
+// }
```

Comment:
we should instead test the new orderInitialPoolDenoms

Reply:
Added the new test case